.travis.yml

```diff
@@ -68,6 +68,22 @@ jobs:
         - docker exec -it pgbuild /bin/bash -c 'cd /build/debug && make licensecheck'
         - "test -z \"`./build/scripts/export_prefix_check.sh`\" || (echo \"functions missing 'ts_' prefix or should be static\"; ./build/scripts/export_prefix_check.sh; exit 1)"
 
+    # This runs tests on ARM32 emulation
+    - if: (type = pull_request) OR (type = cron) OR NOT (branch = master) # run on PRs for debugging todo:: change to pre-release before merge
+      stage: test
+      name: "Regression 9.6 test ARM processors"
+      env:
+        # - PG_VERSION=10.2 PG_GIT_TAG=REL_10_2 ADDITIONAL_CMAKE_FLAGS="-DUSE_DEFAULT_VISIBILITY=1"
+      before_install:
+        - docker run --rm --privileged multiarch/qemu-user-static:register --reset
+        - docker run -t -d --name docker_arm_emulator ngalstyan4/timescaledb-arm:live
```

Comment:
We should move all references to timescale docker hub account, rather than personal